### DISCUSSION TITLE
Can I somehow change the debugger link?

### DISCUSSION THREAD

**Discussion by mkoskl** [symfony/symfony#58967] [Q&A]
Currently, my dev-environment is behind proxy machine and the link I get back contains the IP address of my dev machine, not the domain I use to access the app I'm developing.

Can I somehow change that?

Br, Miika

**Comment by andrew-demb** [comment]
Did you try to configure Symfony according to the info from this link?

https://symfony.com/doc/current/deployment/proxies.html
